<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>二维绘图系统     By 李章月&amp;20221003995</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="MyGraphicsview" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>绘制</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionPolygon"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionSelect"/>
    <addaction name="actionFill"/>
    <addaction name="actionTranslate"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionLine"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionPolygon"/>
   <addaction name="actionSelect"/>
   <addaction name="actionFill"/>
   <addaction name="actionTranslate"/>
   <addaction name="actionDelete"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/line.png</normaloff>:/images/line.png</iconset>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/rectangle.png</normaloff>:/images/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>长方形</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/ellipse.png</normaloff>:/images/ellipse.png</iconset>
   </property>
   <property name="text">
    <string>椭圆形</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/polyline.png</normaloff>:/images/polyline.png</iconset>
   </property>
   <property name="text">
    <string>多边形</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/select.png</normaloff>:/images/select.png</iconset>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/fill.png</normaloff>:/images/fill.png</iconset>
   </property>
   <property name="text">
    <string>填充</string>
   </property>
  </action>
  <action name="actionTranslate">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/move.png</normaloff>:/images/move.png</iconset>
   </property>
   <property name="text">
    <string>平移</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
   </property>
   <property name="text">
    <string>使用帮助</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsview</class>
   <extends>QGraphicsView</extends>
   <header location="global">mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
